prometheus/osfamilymap.yaml

```diff
@@ -206,9 +206,14 @@ RedHat:
           args:
             config.file: /etc/prometheus/blackbox.yml
     repo:
+      {%- if grains.os == 'Amazon' %}
+        {%- set releasever = salt['cmd.run']("rpm -E '%{rhel}'") %}
```

Comment:
Indent 4 spaces, instead of 2 spaces, to line up with the block above (suggestion, not rule).